x-pack/test/functional/apps/saved_objects_management/spaces_integration.ts

```diff
@@ -54,9 +54,9 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       await PageObjects.common.waitUntilUrlIncludes(getSpacePrefix(spaceId));
 
-      expect(await testSubjects.getAttribute(`savedObjects-editField-title`, 'value')).to.eql(
-        'A Pie'
-      );
+      const inspectContainer = await find.byClassName('kibanaCodeEditor');
+      const visibleContainerText = await inspectContainer.getVisibleText();
+      expect(visibleContainerText.includes('A Pie'));
```

Comment:
NIT: ideally we would be parsing the JSON to assert that `attributes.title` is `A Pie`, but that's probably fine.